pachyderm/templates/pachd/deployment.yaml

```diff
@@ -93,12 +93,14 @@ spec:
             fieldRef:
               apiVersion: v1
               fieldPath: metadata.namespace
+        {{- if .Values.pachd.resources }}
```

Comment:
Maybe check for `.Values.pachd.resources.memory` too, just in case someone specifies CPU but not memory?

Reply:
TODO: Add test for this case